caffe-grid/src/test/python/PythonApiTest.py

```diff
@@ -0,0 +1,61 @@
+'''
+Copyright 2016 Yahoo Inc.
+Licensed under the terms of the Apache 2.0 license.
+Please see LICENSE file in the project root for terms.
+'''
+
+from com.yahoo.ml.caffe.CaffeOnSpark import CaffeOnSpark
+from com.yahoo.ml.caffe.Config import Config
+from com.yahoo.ml.caffe.DataSource import DataSource
+from pyspark.sql import DataFrame
+from pyspark.mllib.linalg import Vectors
+from pyspark.sql import Row
+from pyspark import SparkConf,SparkContext
+from itertools import izip_longest
+from pyspark.sql import SQLContext
+import unittest
+
+conf = SparkConf()
+sc = SparkContext(conf=conf)
+sqlContext = SQLContext(sc)
+
+class PythonApiTest(unittest.TestCase):
+    def grouper(self,iterable, n, fillvalue=None):
+        args = [iter(iterable)] * n
+        return izip_longest(fillvalue=fillvalue, *args)
+
+    def setUp(self):
+        #Initialize all objects
+        self.cos=CaffeOnSpark(sc,sqlContext)
```

Comment:
That's required for dataframes. The api signature is old one and we use that in dataframes and notebooks